### PR TITLE
Alpine 3.20

### PR DESCRIPTION
**Pull request by GreyXor**
Hello, this PR adds a variant for Alpine 3.20 and removes support for 3.18.

https://www.alpinelinux.org/posts/Alpine-3.20.0-released.html
https://github.com/alpinelinux/docker-alpine/tree/v3.20

We have to wait for the hub docker update: https://hub.docker.com/_/alpine

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
Depends on https://github.com/docker-library/official-images/pull/16801

**Comment by GreyXor** [comment]
> Depends on [docker-library/official-images#16801](https://github.com/docker-library/official-images/pull/16801)

Thanks, it's now merged and Alpine 3.20 is available on registry https://hub.docker.com/layers/library/alpine/3.20/images/sha256-4a6ffef76277d8c1d912ad489e0c09ddc09fdf7aefae51750a33dc47478c0cc0?context=explore

File: versions.sh [hunk 1]
```diff
@@ -88,7 +88,7 @@ for version in "${versions[@]}"; do
 		bookworm \
 		bullseye \
 		alpine3.19 \
-		alpine3.18 \
+		alpine3.20 \
```
Comment (tianon) [inline versions.sh:91]: This ordering is load-bearing and should be what's in https://github.com/docker-library/php/pull/1514, so I'm going to merge that one instead :bow: :heart:

Thank you for the contribution!

Comment (GreyXor) [reply versions.sh:91]: thanks :+1:

**Pull request closed without merging**